src/picosvg/svg.py

```diff
@@ -910,6 +910,23 @@ def round_floats(self, ndigits: int, inplace=False):
             shape.round_floats(ndigits, inplace=True)
         return self
 
+    def remove_empty_subpaths(self, inplace=False):
+        if not inplace:
+            svg = SVG(copy.deepcopy(self.svg_root))
+            svg.remove_empty_subpaths(inplace=True)
+            return svg
+
+        self._update_etree()
+
+        for idx, (el, (shape,)) in enumerate(self._elements()):
+            if isinstance(shape, SVGPath):
+                shape.remove_empty_subpaths(inplace=True)
+                self._set_element(idx, el, (shape,))
```

Comment:
since the operation on the shape does `inplace=True`, I think it's sufficient to simply iterate over all `svg.shapes()` and run the `shape.remove_empty_subpaths(inplace=True)`, no need to `_set_elelement`. We do something like that for other inplace operations. Or am I missing something else?

Reply:
Hm, I think you are right, will amend.